### ISSUE TITLE
wishlist: update man db after each install/remove

### ISSUE THREAD

**Issue by ofiala-a51** [void-linux/xbps#338]
This is a proposal to update man database (either by running `makewhatis` or otherwise) after each package install or removal, similar to what aptitude does. The reason for this is that it's annoying to run `makewhatis` after each install when I need to search between existing man entries and I simply can't imagine a use case when you would want already removed man entries to show up in your results.

I am not sure about the best way to implement this, which is why I haven't opened a PR, but it could be something as trivial as adding `system("makewhatis")` in all cases where package list is actually modified (if install/remove is successful).

Any feedback is welcome.

**Comment by ericonr** [comment]
I guess implementation is what defines where this wish goes. This could possibly be a part of `xbps-triggers`, in which case it would go in void-packages, or it could be implemented once we have hooks, which is what I would prefer, I think, and in which case I guess it could sit here. This action, however, definitely doesn't fit as something hard coded into the package manager.

**Comment by ofiala-a51** [comment]
You are completely right that it shouldn't be hard-coded, I kinda forgot about xbps-triggers when writing the post 😅

**Comment by ericonr** [comment]
The other complicated thing is how this works with alternate root directories.

**Comment by Duncaen** [comment]
https://github.com/void-linux/xbps/issues/304
